src/main/java/seedu/address/model/cheese/Cheese.java

```diff
@@ -103,7 +109,9 @@ public String toString() {
             .append("; Maturity Date: ")
             .append(getMaturityDate())
             .append("; Expiry Date: ")
-            .append(getExpiryDate());
+            .append(getExpiryDate())
+            .append("; isAssigned: ")
```

Comment:
Just a suggestion (but up to you), you can consider using "Assigned" and "Not Assigned" instead

Reply:
Could you elaborate? 
I was thinking of changing it to status : (Assigned / Not Assigned)

Reply:
yeap something like that